moveit_simple/include/moveit/trajectory_processing/time_optimal_trajectory_generation.h

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
I really don't like this code duplication

Reply:
Me neither. I think it's more like a confirmation that it works as intended. I would suggest pushing moveit_simple to melodic first and then creating a new PR without these.

Reply:
lets just move to master branch now